tools-local/unix/probe-tool.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# Searches the for the specified tool. If tool is not found then, downloads the tool.
+# Paths to search, and download URL is read from the .toolversions file.
+
+if [ -z "$1" ]; then
+    echo "Argument passed as tool name is empty. Please provide a non-empty string."
+    exit 1
+fi
+
+toolName="$1"
+strictToolVersionMatch=0
+
+if [ ! -z "$2" ]; then
+    strictToolVersionMatch="$2"
+fi
+
+scriptpath="$(cd "$(dirname "$0")"; pwd -P)"
+repoRoot="$(cd "$scriptpath/../.."; pwd -P)"
+shellScriptsRoot="$repoRoot/tools-local/unix"
```

Comment:
Yes. Removed.

---
In reply to: [107910838](https://github.com/ravimeda/corefx/pull/2#discussion_r107910838) [](ancestors = 107910838)